h/search/client.py

```diff
@@ -50,6 +51,10 @@ def mapping_type(self):
         """
         return self._mapping_type
 
+    @property
+    def version(self):
```

Comment:
This could use a docstring to indicate that it refers to the Elasticsearch _library_ version, as opposed to the server version.